### PR TITLE
handled update() function of fdb orchagent for FDB FLUSH event

### PR DESCRIPTION
**Pull request by madhanmellanox**
**What I did**
I added code to handle update() function of FDB orchagent for the FDB FLUSH event.

**Why I did it**
I added code because earlier the FDB FLUSH event was not handled not fully, it was saying unsupported. So, I handled the event as like in master branch.

**How I verified it**
Adding a port as VLAN member, learning FDB entries for the port and then removing VLAN membership was causing ERROR logs in SWSS. With this fix, no more error logs were observed.

**Details if related**
Changeset is in orchagent/fdborch.cpp and orechagent/fdborch.h

### CODE REVIEW

**Comment by madhanmellanox** [comment]
retest this please

File: orchagent/fdborch.cpp [hunk 1]
```diff
@@ -307,10 +355,11 @@ void FdbOrch::doTask(Consumer& consumer)
                 }
             }
 
+	    entry.port_name = port;
```
Comment (prsunny) [inline orchagent/fdborch.cpp:358]: Please align

**Review by prsunny — APPROVED** [review]
lgtm. This is directly a port from master, correct?

File: orchagent/fdborch.cpp (hunk 1 above)
Comment (madhanmellanox) [reply orchagent/fdborch.cpp]: Fixed in next commit.

**Comment by madhanmellanox** [comment]
> lgtm. This is directly a port from master, correct?

Yes, this is a direct port from master branch.

**Comment by lguohan** [comment]
can you identify which master branch commit is it coming from?

**Pull request merged**